net-tools/net/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "net"
```

Comment:
how about `netwatch` that is still free on crates.io

Reply:
I haven't pushed the change but I added it in the description. This is exactly the name I had chosen. Push is waiting on taking some stuff out of the crate :tada:

Reply:
great minds think alike 🎉